github/token

```diff
@@ -1,6 +1,21 @@
-#!/bin/bash
+#!/usr/bin/env ruby
+# frozen_string_literal: true
 
-JSON=$(curl -X POST -H "Authorization: Bearer $(./github/jwt)" -H "Accept: application/vnd.github.machine-man-preview+json" "https://api.github.com/installations/$INSTALLATION/access_tokens")
-TOKEN=$(jq -r '.token' <<< $JSON)
+require "json"
+require "net/http"
+require "uri"
 
-echo $TOKEN
+jwt = File.expand_path("jwt", __dir__)
+
+uri = URI("https://api.github.com/installations/#{ENV["INSTALLATION"]}/access_tokens")
+req = Net::HTTP::Post.new(uri)
+req["Authorization"] = "Bearer #{`#{jwt}`}"
```

Comment:
Perhaps execute the `jwt` script elsewhere so we can check if it fails? Or write it in Ruby so we can use it as a library here?